pandas/core/indexes/datetimes.py

```diff
@@ -856,6 +858,10 @@ def date_range(
         Include boundaries; Whether to set each bound as closed or open.
 
         .. versionadded:: 1.4.0
+    unit : str, default None
```

Comment:
Could you add an example in the `Examples` section below?

Reply:
updated+green